src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,67 @@
+package duke;
+
+/**
+ * Translates the user input string into a command object.
+ */
+public class Parser {
+
+    /**
+     * Takes in the user input and converts it into a command.
+     *
+     * @param input the user input string.
+     * @return the command object that represents the action that the user intends to make.
+     */
+    public static Command parse(String input) {
+        if (input.equals("bye")) return new ExitCommand();
```

Comment:
Maybe switch-case would make the code easier to read and more compliant to coding standard :+1:

Reply:
I'm not sure switch case can work here, as the author wants to use the method `.startsWith()` from the `String` object, but it is possible to do some small hacks. I can suggest two possible alternatives here:
- Preserve the if-else blocks, but comply with the coding standard
- You may look at `.split()` method to take the first command and use it into the switch statement